docs/integrations/sources/facebook-marketing.md

```diff
@@ -145,15 +152,12 @@ You can segment the AdInsights table into parts based on the following informati
 For more information, see the [Facebook Insights API documentation.](https://developers.facebook.com/docs/marketing-api/reference/adgroup/insights/)
 
 :::note
-
-Pay attention, that not all fields (e.g. conversions, conversion_values) will be returned for AdInsights, see [docs](https://developers.facebook.com/docs/marketing-api/reference/ads-action-stats/).
-To get all fields You should use custom insights with **breakdowns**.
-
+Please note that not all fields, such as `conversions` and `conversion_values`, will be returned when accessing AdInsights. Refer to the [Facebook documentation](https://developers.facebook.com/docs/marketing-api/reference/ads-action-stats/) for more information. To retrieve all available fields, it is recommended to use Custom Insights and specify the necessary **breakdowns**.
```

Comment:
What is `AdInsights` referring to here? the casing makes it sound like a specific thing like a class or specific stream. Should it be `ad insights`? 

Why would conversions not be returned? That seems weird given that conversions are an important part of ad analytics. Can we give the user a bit more info about when & why this happens? The link didn't provide more clarity for me personally